File: Getting-Practice-with-Python-and-Javascript/2- Control Flow Structures and Data Structures/Data Structures/Python/Excercises/program-6.py
# 6-Crear un conjunto con los números divisibles por 3 del 1 al 10 y mostrar el número de elementos en el conjunto.

conjuntoNumeros = []

for x in range(1,10):
    if x % 3 == 0:
        conjuntoNumeros.append(x)


print(len(conjuntoNumeros))

File: Getting-Practice-with-Python-and-Javascript/2- Control Flow Structures and Data Structures/Data Structures/Javascript/Excercises/program-6.js
// 6-Crear un conjunto con los números divisibles por 3 del 1 al 10 y mostrar el número de elementos en el conjunto.

let conjuntoNumeros = [];

for (let i = 1; i < 10; i++) {
    if (i % 3 == 0) {
        conjuntoNumeros.push(i)
    }
}

console.log(conjuntoNumeros)